tutorials/01_manual_input.tex

```diff
@@ -18,24 +18,27 @@ \subsection*{Subjects covered}
 \subsection*{Example effects table}
 The data that you are going to enter in this tutorial is taken from the effects table for lixisenatide as included in the \href{https://www.ema.europa.eu/documents/template-form/day-80-assessment-report-overview-d120-loq-template-guidance-rev-1019_en.docx}{EMA Guidance document for critical assessment reports} (see page 72).  For didactic purposes, we are going to work with a simplified version of that table which includes HbA1c as a favourable effect and nausea as an unfavourable effect (Figure \ref{fig:lixisenatide_effect_table}).
 
-\begin{figure}[!h]
+\begin{figure}[!ht]
     \centering
     \includegraphics[width=\textwidth]{fig/lixiEffectTable.png}
     \caption{Simplified effects table for lixisenatide, a treatment for type 2 diabetes mellitus}
     \label{fig:lixisenatide_effect_table}
 \end{figure}
 
-\subsection*{Sign in to MCDA.drugis.org}
-\leftpointright \, Open your browser and navigate to \href{https://mcda.drugis.org}{https://mcda.drugis.org}. Use your Google account to sign in. You will be redirected to your personal homepage, containing your previously created workspaces, both finished and unfinished (if any).
+\subsection*{Sign in to your organisation's ADDIS/MCDA.}
+\leftpointright \, Open your browser and navigate to the URL where your organisation provides ADDIS/MCDA.
```

Comment:
This is a tough line since it only applies to closed. Suggestion: "Open your browser and navigate to ADDIS/MCDA; the public version (https://mcda.drugis.org) or the enterprise version (URL provided by your organisation)"

Reply:
or just split into two statements and sidebar the closed one